README.md

```diff
@@ -1 +1,39 @@
-# Kaidan
+# Kaidan - Cross plattform xmpp client
+
+Installation
+------------
+
+On Debian do the following:
+
+Create a working directory
+
+ * mkdir src
+ * cd src
+
+Fetch swift source
+
+ * wget https://github.com/swift/swift/archive/swift-3.0.zip
```

Comment:
@MRZA-MRZA If you want to change the name, just use `-O <the name you want>` in wget